Tests/SpeziFoundationTests/MainActorExecutionTests.swift

```diff
@@ -15,17 +15,49 @@ import Testing
 /// Internal helper actor to be able to have code run guaranteed off the main actor (by scheduling it onto a background queue)
 @globalActor
 private actor TestActor: GlobalActor {
+    final class DispatchQueueExecutor: SerialExecutor {
```

Comment:
Is this type actually needed here? I might miss something, but if we don't use the tests on Linux, we could stick with the old setup?